### PR TITLE
feat: webpack bundling working

### PR DESCRIPTION
**Pull request by d-jeffery**
## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: client/src/extension.browser.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (rhamzeh) [inline client/src/extension.browser.ts:1]: Let's remove these Microsoft headers

File: client/src/extension.browser.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+			language: "json"
+		});
+
+		return (await window.showTextDocument(doc)).document;
+	});
+
+	context.subscriptions.push(transformCommand);
+
+
+	//context.subscriptions.push(disposable);
```
Comment (rhamzeh) [inline client/src/extension.browser.ts:50]: Can we delete this if it is not needed?

File: client/src/test/index.browser.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+require('mocha/mocha'); // import the mocha web build
```
Comment (rhamzeh) [inline client/src/test/index.browser.ts:1]: Can this use the import syntax instead or does it have to be require?

File: server/src/server.common.ts [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+		hasConfigurationCapability = !!(
+			capabilities.workspace && !!capabilities.workspace.configuration
+		);
+		hasWorkspaceFolderCapability = !!(
+			capabilities.workspace && !!capabilities.workspace.workspaceFolders
+		);
+		hasDiagnosticRelatedInformationCapability = !!(
+			capabilities.textDocument &&
+			capabilities.textDocument.publishDiagnostics &&
+			capabilities.textDocument.publishDiagnostics.relatedInformation
```
Comment (rhamzeh) [inline server/src/server.common.ts:46]: Can you make sure that this is returning true?

File: server/src/server.common.ts [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+
+	connection.onDidChangeConfiguration(_change => {
+		// Revalidate all open text documents
+		documents.all().forEach(validateTextDocument);
+	});
+
+
+	// Only keep settings for open documents
+	// documents.onDidClose(e => {
```
Comment (rhamzeh) [inline server/src/server.common.ts:88]: Delete

File: client/src/test/index.browser.ts (hunk 3 above)
Comment (d-jeffery) [reply client/src/test/index.browser.ts:1]: Just following docs: https://code.visualstudio.com/api/extension-guides/web-extensions#web-extension-tests

Changing it, but may have to revert if it doesnt work with the test suite im trying to get working.

File: server/src/server.common.ts (hunk 4 above)
Comment (d-jeffery) [reply server/src/server.common.ts:46]: True for node & web versions when running from launch.

**Review by rhamzeh — APPROVED** [review]

**Pull request merged**